tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -92,6 +94,34 @@ describe('Discover', function() {
       expect(wrapper.find('NewQuery')).toHaveLength(0);
       expect(wrapper.find('EditSavedQuery')).toHaveLength(1);
     });
+
+    it('handles navigating to new date', async function() {
```

Comment:
This attempts to test what happens when browser back button is pressed.

Reply:
👍